selfdrive/car/car_helpers.py

```diff
@@ -8,12 +12,13 @@
 from selfdrive.swaglog import cloudlog
 import cereal.messaging as messaging
 from selfdrive.car import gen_empty_fingerprint
+from selfdrive.controls.lib.events import Alert
 
 from cereal import car
 EventName = car.CarEvent.EventName
 
 
-def get_startup_event(car_recognized, controller_available, fw_seen):
+def get_startup_event(car_recognized: bool, controller_available: bool, fw_seen: bool) -> Dict[str, Alert]:
```

Comment:
Shouldn't this return a `car.CarEvent.EventName`? Why is mypy not catching this?

Reply:
Good catch, I've got the wrong return type. mypy won't allow EventName as a type, but all paths result in an int being returned so I set it to that.

I'm unsure as to why mypy missed this. I'll look into that a bit deeper.